test/view_component/integration_test.rb

```diff
@@ -214,12 +215,28 @@ class IntegrationTest < ActionDispatch::IntegrationTest
     assert_select "title", "Component Previews"
   end
 
+  test "test preview index renders custom application layout if configured" do
+    ViewComponent::Base.default_preview_layout = "admin"
+    get "/rails/view_components"
+    ViewComponent::Base.default_preview_layout = nil
+
+    assert_select "title", "ViewComponent - Admin - Test"
```

Comment:
Is it time for us to write a helper method for this pattern that makes it clearer we're asserting the right layout? That we're proxying the laying being applied by asserting the page title is sort of a weird detail. 
```suggestion
    assert_admin_layout
```

?